src/components/datePicker/tests/DatesSelectionConnected.spec.tsx

```diff
@@ -191,10 +195,21 @@ describe('Date picker', () => {
       expect(_.findWhere(store.getState().datePickers, { id: DATES_SELECTION_ID }).lowerLimit).toBe(expectedValue);
     });
 
-    it('should change the upper limit in the state when calling the onChange prop with the upper limit', () => {
+    it('should change the upper limit in the state to the new lower limit date when calling the onChange prop with the lower limit when isRange prop is false', () => {
+      mountComponent({isRange: false});
       let expectedValue: Date = new Date(new Date().setHours(5, 5, 5, 5));
 
-      datesSelection.props().onChange(expectedValue, true);
+      datesSelection.props().onChange(expectedValue, false);
+
+      expect(_.findWhere(store.getState().datePickers, { id: DATES_SELECTION_ID }).upperLimit).toBe(expectedValue);
+    });
+
+    it('should not change the upper limit in the state when calling the onChange prop with the lower limit when isRange prop is true', () => {
+      mountComponent({isRange: true});
+      let newValue: Date = new Date(new Date().setHours(5, 5, 5, 5));
```

Comment:
Does it still work if you change for a const? I know that sometimes `moment` has problems with read only dates.